### PR TITLE
[nnfwapi] Test : Add "Add" operation tests

### PR DESCRIPTION
**Pull request by wateret**
Add "Add" operation tests - 1 positive / 3 negative.

ONE-DCO-1.0-Signed-off-by: Hanjoung Lee <hanjoung.lee@samsung.com>

### CODE REVIEW

**Review by hyunsik-yoon — DISMISSED** [review]
LGTM

File: runtime/onert/frontend/base_loader/include/base_loader.h [hunk 1]
```diff
@@ -235,8 +235,8 @@ BaseLoader<LoaderDomain>::BaseLoader::convertActivation(const ActivationFunction
     case ActivationFunctionType::ActivationFunctionType_TANH:
       return ir::Activation::TANH;
     default:
-      throw std::runtime_error(std::string("Unsupported activation type: ")
-                                   .append(EnumNameActivationFunctionType(type)));
+      throw std::runtime_error(std::string("Unsupported or invalid activation type: ") +
+                               std::to_string(static_cast<int>(type)));
```
Comment (wateret) [inline runtime/onert/frontend/base_loader/include/base_loader.h:239]: Added a fix - `EnumNameActivationFunctionType()` fails for invalid `type`.

**Review by hyunsik-yoon — APPROVED** [review]
LGTM
